lib/TWCManager/Vehicle/TeslaAPI.py

```diff
@@ -1268,7 +1268,7 @@ def wakeVehicle(self, vehicle):
                     return self.wakeVehicle(vehicle)
             elif req.status_code == 429:
                 # We're explicitly being told to back off
-                self.errorCount = max(30, self.errorCount)
```

Comment:
Removing this line is a bad idea:

`updateCarApiLastErrorTime` only increases errorCount by 1. When the errorCount is only 1, `getCarApiErrorRetryMins` will return 0 and there will be no backoff.

But if we get a 429 here, we already hit the rate limit, so we know that we need to delay the next API call.

Setting errorCount to 30 makes sure we wait 10 minutes (because of `return min(errorCount, 10)` in `getCarApiErrorRetryMins`).

Reply:
Ah, good call. I'll fix that.